daisy/prow_images/functional_tests/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM debian:stretch
+
+ENV GOOGLE_APPLICATION_CREDENTIALS /etc/service-account/service-account.json
+ENV GCLOUD_VERSION 168.0.0
+ENV PATH "/google-cloud-sdk/bin:${PATH}"
+
+RUN apt-get update
+RUN apt-get upgrade -y
+RUN apt-get install -y \
+    git \
+    golang \
```

Comment:
Don't use the package, it's crazy outdated. Pull the tar and install (assuming a golang container wont work, installing golang might be easier that installing python)